### PR TITLE
Added multiply to torch frontend

### PR DESCRIPTION
**Pull request by guitld**

### CODE REVIEW

**Comment by guitld** [comment]
This PR reffers to  issue #14573

File: ivy/functional/frontends/torch/tensor.py [hunk 1]
```diff
@@ -991,3 +991,7 @@ def fill_(self, value):
 
     def nonzero(self):
         return torch_frontend.nonzero(self)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16",)}, "torch")
+    def multiply(self, other, *, out=None):
+        return torch_frontend.multiply(self._ivy_array, other, out=out)
```
Comment (YushaArif99) [inline ivy/functional/frontends/torch/tensor.py:997]: I don't think there's a need to do `self._ivy_array` here. Simply calling the frontend function by passing `self` should work as the frontend function's decorators will take care of converting the inputs to and from `ivy arrays` to `frontend array` etc.

File: .idea/ivy.iml
Comment (YushaArif99) [inline .idea/ivy.iml:1]: This file should not be modified

File: .idea/misc.xml
Comment (YushaArif99) [inline .idea/misc.xml:1]: same for this file as well

File: .idea/vcs.xml
Comment (YushaArif99) [inline .idea/vcs.xml:1]: same for this file as well

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
Hi @guitld 

Thanks for making the PR. Could you kindly address the abovementioned points? Also, there are a couple of merge conflicts that will need to be resolved. Could you kindly also resolve them as this will allow the CI tests to run? Once all the relevant tests are passing, you can re-request a review 🙂

**Comment by guitld** [comment]
Hi @YushaArif99 , just commited the changes you suggested! Thx for the review :)
I don't know why it's failing in the check formatting test though

**Review by YushaArif99 — CHANGES_REQUESTED** [review]
HI @guitld 

The `.idea` files are still showing in the modified files. You can try using `git revert` to revert the commit that modified these files. If that doesnt work, one solution could be to reset your HEAD to point to the master branch using `git reset --hard origin`. This will reset your branch's HEAD to point to the master branch and so your local changes would get removed. Its best to keep a backup before you use `git reset`. Once this is done, you can then do the usual `git add->git commit->git push` to create a new commit. **NOTE:** its best to manually add the files using `git add <file path>` rather than `git add .` as the latter might add unwanted files (eg: the `idea` files) to the staging area.

Hope that helps 🙂

**Comment by guitld** [comment]
Hi @YushaArif99

I think .idea is not modified anymore. Also, the conflicts are resolved.
Could you review it again please? :)

**Review by YushaArif99 — APPROVED** [review]
Hi @guitld 

Thanks for removing the `.idea` files. The CI tests are also passing now which means we can merge this PR 🙂

**Pull request merged**